README.md

```diff
@@ -1,21 +1,92 @@
 # dockstore-tool-star
+Versioned Docker image and CWL tools for the STAR Read Aligner
 
-Dockerized STAR and CWL tool definition for Dockstore.
+# Description
+This repository contains a dockerfile and the following CWL tools:
 
-## Contributing
+- [star_index.cwl](cwl/star_index.cwl): indexes the reference genome using the `STAR --runMode genomeGenerate` command
+- [star_align.cwl](cwl/star_align.cwl): aligns paired-end reads to the indexed reference genome using the `STAR --runMode alignReads` command  
+- [star_align-se.cwl](cwl/star_align-se.cwl): aligns single-end reads to the indexed reference genome using the `STAR --runMode alignReads` command
+- [combine_counts_study.cwl](cwl/combine_counts_study.cwl): combines gene counts produced by multiple samples into a single table. The script can be found [here](bin/combine_counts_study.R)
 
-New features should be opened on branches from `develop` and merged there with a code review.
+Documentation for the STAR Aligner functions used can be found [here](https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf)
 
-CWL definitions should go in [/cwl](cwl/).
+## Dockerfile
```

Comment:
Everything from here down is confusing because it reads like the instructions to set up tool repository, not a description of this tool repository. Instead of "A Dockerfile should exist", if you keep that section it should be "A Dockerfile exists..."

But what I'd really recommend is to rewrite this slightly. Re-add `#Contributing` above all that follows and you'll have to rewrite it less.